openml-lightgbm-module/openml-lightgbm/pom.xml

```diff
@@ -14,38 +14,39 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>com.feedzai.openml.lightgbm.meta</groupId>
-        <artifactId>openml-lightgbm-meta-module</artifactId>
+        <groupId>com.feedzai.openml.lightgbm</groupId>
+        <artifactId>openml-lightgbm-module</artifactId>
         <version>0.0.0-SNAPSHOT</version>
     </parent>
 
     <groupId>com.feedzai</groupId>
     <artifactId>openml-lightgbm</artifactId>
     <version>${project.version}</version>
 
-    <description>Openml Microsoft LightGBM Machine Learning Model and Classifier provider</description>
+    <description>OpenML Microsoft LightGBM Machine Learning Model and Classifier provider</description>
+
+    <properties>
```

Comment:
Pull the properties back to the lightgbm-module so we can read the lightgbmlib.version property from it and guarantee it doesn't go out of sync instead of maintaining it in two places.

Reply:
You are right, since both submodule are using lightgbmlib, I will move it to parent module